messages/reporter.md

```diff
@@ -1,3 +0,0 @@
-# bulkV2Result
-
-Job %s | Status %s | Records processed %d | Records failed %d
```

Comment:
unused msg so I removed the whole file, left over from previous PR where we refactored bulk commands to use oclif/multi-stage-output.